### DISCUSSION TITLE
list the dependencies

### DISCUSSION THREAD

**Discussion by efa** [ra3xdh/qucs_s#913] [Q&A] [answered]
in the installation page:
https://github.com/ra3xdh/qucs_s/wiki/Installation
you should list the dependencies. On Debian are at least:
$ sudo apt install libqt6core6 libqt6dbus6 libqt6gui6 libqt6network6 libqt6printsupport6 libqt6svg6 libqt6svgwidgets6 libqt6widgets6 libts0 qt6-gtk-platformtheme qt6-qpa-plugins qt6-translations-l10n

**Comment by efa** [comment]
anyway good that you generate the AppImage too for the Continuous build

**Comment by ra3xdh** [comment, marked as answer]
I have just updated the wiki page. Thanks for pointing this.
